bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1290 @@
+				<label>Sleep</label>
+				<description>Set sleep timer in cycles</description>
+				<required>false</required>
+				<default>5</default>
+			</parameter>
+
+			<parameter name="secureSend" type="boolean">
+				<label>Secure Send</label>
+				<description/>
+				<required>false</required>
+				<default>true</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="t18">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T18</label>
+		<description>Simple Light with feedback </description>
+
+		<channels>
+			<channel id="onOff" typeId="onoff"/>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+		<config-description>
+			<parameter name="node" type="integer" min="1" max="256">
+				<label>Node</label>
+				<description>Node</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="slot" type="integer" min="1" max="256">
+				<label>Slot</label>
+				<description>Slot</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="sleep" type="integer" min="1" max="1000">
+				<label>Sleep</label>
+				<description>Set sleep timer in cycles</description>
+				<required>false</required>
+				<default>5</default>
+			</parameter>
+
+			<parameter name="secureSend" type="boolean">
+				<label>Secure Send</label>
+				<description/>
+				<required>false</required>
+				<default>true</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="t19">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="gateway"/>
+		</supported-bridge-type-refs>
+		<label>T19</label>
+		<description>Single Color LED Strip</description>
+
+		<channels>
+			<channel id="onOff" typeId="onoff"/>
+			<channel id="rollerBrightness" typeId="rollerBrightness"/>
+			<channel id="dimmerBrightness" typeId="dimmerBrightness"/>
+			<channel id="sleep" typeId="sleep"/>
+			<channel id="lastStatusStored" typeId="lastStatusStored"/>
+			<channel id="healthy" typeId="healthy"/>
+		</channels>
+
+		<config-description>
+			<parameter name="node" type="integer" min="1" max="256">
+				<label>Node</label>
+				<description>Node</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="slot" type="integer" min="1" max="256">
+				<label>Slot</label>
+				<description>Slot</description>
+				<required>true</required>
+			</parameter>
+			<parameter name="sleep" type="integer" min="1" max="1000">
+				<label>Sleep</label>
+				<description>Set sleep timer in cycles</description>
+				<required>false</required>
+				<default>5</default>
+			</parameter>
+			<!-- To Implement -->
+			<!-- <parameter name="secureSend" type="boolean"> -->
+			<!-- <label>Secure Send</label> -->
+			<!-- <description></description> -->
+			<!-- <required>false</required> -->
+			<!-- <default>true</default> -->
+			<!-- </parameter> -->
```

Comment:
If this isn't implemented you could remove it and stash it via Git to keep it for future implementations. Please check all.

Reply:
removed.